core/trino-main/src/main/java/io/trino/sql/planner/planprinter/JsonRenderer.java

```diff
@@ -19,6 +19,8 @@
 import io.trino.sql.planner.plan.PlanFragmentId;
```

Comment:
How do the `EXPLAIN` look now for more complex plans? Aren't they too verbose?

Reply:
Do you mean JSON or text plan? 

For the text plan not much has changed, only the identifier part which is much more readable now IMO. For instance `LimitPartial[10]` -> `LimitPartial[count = 10]`

Reply:
Please add example of text plan if commit description (e.g. with some scan and join)

Reply:
Won't that make the commit too long? Instead, I've added it as a PR comment.